### ISSUE TITLE
Class session.store does not exist !? 

### ISSUE THREAD

**Issue by tahashieenavaz** [vluzrmos/collective-html#5]
Hello :blush: 
 I wanted to use this but when i try to user Form i give this error
`
ErrorException in Container.php line 738:
Class session.store does not exist (View: C:\xampp\htdocs\Portal\blog\resources\views\master.blade.php) (View: C:\xampp\htdocs\Portal\blog\resources\views\master.blade.php)
`

**Comment by vluzrmos** [comment]
Maybe you need to active the laravel/lumen SessionServiceProvider.

Try install it:

```
composer require illuminate/session
```

and active it on your bootstrap/app.php file:

```
$app->register(Illuminate\Session\SessionServiceProvider::class);
```

You can see that service "session.store" being activated on that file:

https://github.com/illuminate/session/blob/master/SessionServiceProvider.php#L42

By default it was removed from Lumen 5.2 because it is now focused on stateless apps (apps without sessions).

**Comment by vluzrmos** [comment]
Fixed on version v1.0.11
